Week 1/Storage Services.md

```diff
@@ -0,0 +1,113 @@
+# Week 1: Storage Services
+---
+## **Mục Lục**
+
+### I. Phân biệt File, Block và Object storage
+- #### 1. File storage là gì ?
+- #### 2. Block storage là gì ?
+- #### 3. Object storage là gì ?
+
+### II. Giải pháp của các vendors để đáp ứng 3 dịch vụ trên 
+
+### III. Ceph - Giải pháp SDS đáp ứng 3 dịch vụ trên
+### IV. Tài liệu tham khảo
+
+
+---
+### I. Phân biệt File, Block và Object storage
+**File**, **blocks** và **objects** là những formats lưu trữ chứa, sắp xếp và thể hiện dữ liệu theo những cách khác nhau, với những chức năng và giới hạn tiêng. File storage sắp xếp và thể hiện dữ liệu theo cấp bạc (hierachy) của files trong các folders. Block storage sẽ chia dữ liệu theo các khối và object storage sẽ quản lý dữ liệu và liên kết nó với metadata tương ứng
+#### 1. File storage là gì ?
+**File Storage**: Dữ liệu được chia vào các folders. **File storage** sử dụng cấu trúc cấp bậc để sắp xếp file theo các folders và folders con theo ý người dùng
+VD: Lưu file trong máy tính chính là sử dụng **file storage**
+Khi cần truy cập vào dữ liệu, thì máy tính sẽ cần biết đường dẫn đến dữ liệu đó. Khi 1 file được lưu trữ theo cách này, sẽ có những metadata được gán theo nó như creation date, modification date và file size.
+Files có thể được lưu trữ trong các thiết bị NAS, hệ thống lưu trữ nền cloud, network drives, computer hard drives và flash drives
```

Comment:
Chỗ này e chưa chỉ rõ ra được cái điểm khác lớn nhất của file storage là gì, dẫn tới ưu nhược điểm bên dưới toàn những ý chung chung không có gì nội trội cả, use-case cũng chưa rõ ràng